### ISSUE TITLE
Dockerfile: no buildable Go source files in /root/gokahing/goofys

### ISSUE THREAD

**Issue by cekvenich** [kahing/goofys#309]
My docker was not happy. So I used 
https://raw.githubusercontent.com/kahing/goofys/master/bench/Dockerfile

It fails:

Step 13/14 : RUN go get . && make install
 ---> Running in 3f062786495e
can't load package: package github.com/kahing/goofys: no buildable Go source files in /root/gokahing/goofys
The command '/bin/sh -c go get . && make install' returned a non-zero code: 1
[ec2-user@ip-172-31-91-98 ~]$ can't load package: package github.com/kahing/goofys:
can't load package: package github.com/kahing/goofys: no buildab le Go source files in /root/gokahing/goofys

**Comment by kahing** [comment]
How are you building the image?

**Comment by cekvenich** [comment]
I download said dockerfile and build.

**Comment by kahing** [comment]
What's the exact command that you used and which directory you executed it in? It needs to be run from the goofys source directory

**Comment by cekvenich** [comment]
Oh
 I just used the dockerfile in a blank folder, let me try again.
--
  vic@eml.cc

**Comment by cekvenich** [comment]
![goof](https://user-images.githubusercontent.com/22248870/38959562-1a28ebb6-432f-11e8-952b-5ecec0e919f0.PNG)
 
It fails when run form folder.
I do git clone and go to bench folder and run docker build .
That is the error attached. 

Honestly, I just want a working docker image that I can use to build on. The ones listed I can't tell what they are and what they have.

**Comment by kahing** [comment]
you need to run from the base directory:

```
goofys: docker build -f bench/Dockerfile .
```

**Comment by cekvenich** [comment]
That did build it. But on docker run it says:
RioFS password file ~/.passwd-riofs missing

I am not looking for RioFS.

**Comment by cekvenich** [comment]
OK. I just made a blank docker w/ goofys ( https://hub.docker.com/r/nbake/nbake/ )

THX!
